README.md

```diff
@@ -0,0 +1,38 @@
+# Some Engineering Helm Chart Repository
+
+![](https://user-images.githubusercontent.com/2124094/164599444-448f92f1-7a73-4a86-a377-bc650f10e756.png)
+
+## Usage
+
+```bash
+$ helm repo add someengineering https://helm.some.engineering
+```
+
+## Charts
+
+### `resoto`
+
+- [Chart documentation](./someengineering/resoto/README.md)
+- [Installation instructions](https://resoto.com/docs/getting-started/install-resoto/kubernetes)
```

Comment:
Should we also embed installation instructions in here?

Reply:
I don't think it's worth the management overhead. We should not be maintaining two copies of the install instructions.